src/stores/rootStore.ts

```diff
@@ -167,15 +167,16 @@ export const useRootStore = defineStore('root', {
 
     /**
      * This is used to show Legal Obligations only for a new business
-     * that has no tasks and hasn't filed anything yet (except their IA).
+     * that has no tasks and hasn't filed anything yet (except their application).
      **/
     isBusinessWithNoMaintenanceFilings (state: RootStateIF): boolean {
       const filingHistoryListStore = useFilingHistoryListStore()
       return (
         // no todo items
         (state.tasks.length === 0) &&
-        // only the IA or Registraion filing history item
+        // only the Amalgamation or IA or Registration filing history item
         (filingHistoryListStore.getFilings.length === 1) && (
+          EnumUtilities.isTypeAmalgamation(filingHistoryListStore.getFilings[0]) ||
```

Comment:
This is to SHOW the Legal Obligations panel for a new TED.

Before:
![image](https://github.com/bcgov/business-filings-ui/assets/10002889/73eab0d1-2c49-49f4-a941-dcfb4e71207c)

After:
![image](https://github.com/bcgov/business-filings-ui/assets/10002889/1b95b4b2-43a0-425a-9b37-432fc417767c)